### PR TITLE
Graphics!

### PR DESCRIPTION
**Pull request by mikolajmirko**
Including avatar widget (will be used in form and places).

### CODE REVIEW

**Comment by mikolajmirko** [comment]
Comments:

- GraphicAsset includes string _label_ for potential description in image picker form widget and accessibility tags.
- _badges_ package installed for future use (I was having some fun with it already)

File: lib/utils/icon_sets.dart [hunk 1]
```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+
+enum GraphicAssetType { childAvatars, awardsIcons, badgeIcons }
+
+class GraphicAsset {
+	final String filename;
+	final String label;
```
Comment (stasgora) [inline lib/utils/icon_sets.dart:7]: I assume that will be used for filtering later? If so can we make it an enum?

Comment (stasgora) [reply lib/utils/icon_sets.dart:7]: I know I know, we probably want a different enum for each category. Well, we could either extend GraphicAsset three times, or do that:
image
Either way we need a cast to access the label value

Comment (mikolajmirko) [reply lib/utils/icon_sets.dart:7]: Hmm, but why splitting it? Can't we have just one GraphicLabel enum with labels? (If you have string so much 😆) They probably will never not overlap and even if - there will be no duplicates in one GraphicType.

`enum GraphicLabel  { boy, girl, gift, relax, activity (...) }`

Comment (stasgora) [reply lib/utils/icon_sets.dart:7]: My point was that with enums we can:
- enumerate (daaa)
- get index
- get name
which simplifies things if we would like for example to show a combo box that filters the visible icons (and there are many). If we had one enum for all categories we wouldn't know which part of it to show in such ui element.

Comment (mikolajmirko) [reply lib/utils/icon_sets.dart]: Ok, didn't think of it as categories - just labels.. like on tooltips.
Changed applied.

Comment (stasgora) [reply lib/utils/icon_sets.dart]: When labels start to repeat they become categories ;)

**Pull request merged**